content/synthetics/browser_check.md

```diff
@@ -0,0 +1,75 @@
+
+## Overview
+
+Browser Checks are scenarios executed by Datadog on your web applications. They run at configurable periodic intervals from multiple locations around the world, and from multiple devices. These checks verify both that your applications are up and responding to requests, and that any conditions defined in your scenarios are met.
+
+## Configuration
+
+### Test details
+
+Define the configuration of your Browser Check.
+
+{{< img src="synthetics/browser_check/browser_check_configuration.png" alt="Browser Check make request" responsive="true" style="width:80%;">}}
+
+1. Give your Browser Check a name. 
+2. Enter the Start URL. This is the URL from which your Browser Check starts the scenario.
+3. Define on which device to run your check. Available devices are: `Laptop Large`, `Tablet`, and `Mobile Small`.
+4. Pick-up locations to run the test from. Available locations are:
+    * Frankfurt (Request made from an AWS Datacenter)
+    * Tokyo (Request made from an AWS Datacenter)
+5. Choose a Check frequency between "1 run per 5 minute interval" to "1 run per week":
+
+    {{< img src="synthetics/browser_check/check_frequency.png" alt="Check frequency" responsive="true" style="width:80%;">}}
+
+### Notifications
+
+A notification is sent if at least one step of the Browser Check scenario fails. To configure your notifications:
+
+{{< img src="synthetics/browser_check/browser_check_notification.png" alt="Browser check notification" responsive="true" style="width:80%;">}}
+
+1. Select users and/or [services][1] to send the notifications to.
+2. Enter a **message** for the Browser Check. This field allows standard [Markdown formatting][2]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+3. Click **Save Details and Record Test** to save your Browser Check.
+4. Start to record your test.
+
+## Record test
+
+**Tests can be only recorded from [Google Chrome][3]. To record your test, download the [Datadog Record Test extension for Google Chrome][4].**
+
+
+{{< img src="synthetics/browser_check/browser_check_record_test.png" alt="Browser check record test" responsive="true" >}}
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your Browser Check.
+3. Your actions are recorded and used to create steps within your Browser Check Scenario.
+4. Optionally, use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_check/browser_check_assertions.png" alt="Browser Check assertions" responsive="true" style="width:80%;">}}
```

Comment:
Could we detail the actions a bit more? Especially the Variable action which is pretty useful (to fill forms for instance). [By clicking on the hand and then selecting a field on the page, we're able to assign it the variable OR by selecting Use Variable in the Type text action.](https://cl.ly/642ddd626e07)

Reply:
Also there is nothing on the different types of assertions, hover and navigate steps, is that on purpose?